app/views/bikes/edit.html.slim

```diff
@@ -0,0 +1,20 @@
+=simple_form_for @bike, :url => { :controller => :bikes, :action => :update } do |f|
+  =image_tag @bike.avatar_url, width:"200px"
+  br
+  =f.file_field :avatar
```

Comment:
equal code detected. 
you have 3 same forms. please move same code in separate partial.